isp/convenience.go

```diff
@@ -209,27 +209,6 @@ func NewWithClientCredentials(clientID, clientSecret, organization string) *High
 	}
 }
 
-// NewWithOktaClientCredentials creates a new authenticated client using the OAuth
-// 2.0 client credentials flow, caching and refreshing the access token as
-// needed whenever requests to the API are made. Leave the organization blank
-// if using multi-org tokens. It is specific to WBD-Okta and replaces NewWithClientCredentials.
-func NewWithOktaClientCredentials(clientID, clientSecret string) *HighLevelClient {
```

Comment:
Looks like you might need to pull/merge from `main`, this just merged a bit ago and definitely should not be removed.